Update definition of menu_bar (#44)
rl-institut/open_plan: docs/view_components/menu_bar.rst

Actions
^^^^^^^

> * Clicking on any of the menus' option results in a drop-down list of sub-menu's options

Reviewer: I like these changes

> * Clicking on any of the menus' option results in a drop-down list of sub-menu's options

Reviewer: Hello

> * ysdkljasdkasldjasldkad alkd

Reviewer 2: Should be ;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv

Author: ```suggestion
;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv
```

> * ysdkljasdkasldjasldkad alkd

Reviewer 3: ```suggestion
* sdfsdfsdfsdfsdfsdfsdf
```

> ;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv

Author: wssdfsf

Author: askjdsadkah

Reviewer: hello

Author: ```suggestion
;askdjf;lkadhf;lkadjhfl;ahjdgo;
```

* Clicking on *File* would show a drop-down list with the following submenus or options:
    I. Load project

>     II. New project

Reviewer 4: Hallo Test

    III. Save project
    IV. Save project as
    V. Export project
    VI. Exit open_plan

> * clicking on *Preferences* would show aadasdsaddasdd ist with the following submenus or options:

Reviewer 5: Dropdown lists are not handy 
```suggestion
* clicking on *Preferences* would show pop-up window with the following submenus or options:
```

Rendering of the view-component
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> * Rectanguladasdasdadadadada above the flow chart ribbon

Reviewer 3: ```suggestion
* above the flow chart ribbon
```
